### PR TITLE
add loader3

### PR DESCRIPTION
**Pull request by YarynaN**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -61,18 +61,29 @@ h1 {
...
-    height: 32px;
-    border-radius: 50%;
-    border-top: 3px solid rgba(255, 255, 255, 0.5);
-    border-right: 3px solid rgba(255, 255, 255, 0.5);
-    border-bottom: 3px solid rgba(255, 255, 255, 0.5);
-    border-left: 3px solid rgba(255, 255, 255, 1);
-    animation: loader 1s cubic-bezier(.41, .25, .32, .83) infinite;
+    filter: blur(1.3px);
+}
+.loader-3_item-1 {
```
Comment (YarynaN) [inline css/style.css]: I suggest keeping naming consistent (kebab case). e.g.`loader-3-item-1`

File: css/style.css [hunk 2]
```diff
@@ -61,18 +61,29 @@ h1 {
...
+    position: absolute;
+    background: white;
+    border-radius: 100%;
+    opacity: 0;
+    width: 60px;
```
Comment (YarynaN) [inline css/style.css]: Could you please change size to 40px? It will look slightly better.

File: css/style.css (hunk 1 above)
Comment (nikondotlg) [reply css/style.css]: fixed to kebab-case

File: css/style.css (hunk 2 above)
Comment (nikondotlg) [reply css/style.css]: changed items size to 40px

**Review by andriybil — APPROVED** [review]

**Review by OleksiiSolomakha — APPROVED** [review]

**Review by EvaMalinina — COMMENTED** [review]
Good job!

**Pull request merged**